file/file_util.cc

```diff
@@ -151,14 +151,21 @@ IOStatus GenerateOneFileChecksum(
         requested_checksum_func_name +
         " from checksum factory: " + checksum_factory->Name();
     return IOStatus::InvalidArgument(msg);
+  } else {
+    // For backward compatibility, requested_checksum_func_name can be empty.
```

Comment:
Maybe add a sentence of comments on line 128 to indicate if requested function name does not match the one from factory, IOStatus::InvalidArgument is returned

Reply:
Good idea, thanks. Done.